ballerina/tests/generated_client.bal

```diff
@@ -197,6 +210,36 @@ client class RainierClient {
         return 'object;
     };
 
+    isolated resource function get orderItems() returns stream<OrderItem, error?> {
+        stream<record{}, sql:Error?>|Error result = self.persistClients.get(ORDER_ITEM).runReadQuery(OrderItem);
+        if result is Error {
+            return new stream<OrderItem, Error?>(new OrderItemStream((), result));
+        } else {
+            return new stream<OrderItem, Error?>(new OrderItemStream(result));
+        }
+    };
+
+    isolated resource function get orderItems/[string orderId]/[string itemId]() returns OrderItem|error {
+        return (check self.persistClients.get(ORDER_ITEM).runReadByKeyQuery(OrderItem, {orderId: orderId, itemId: itemId})).cloneWithType(OrderItem);
+    };
+
+    isolated resource function post orderItems(OrderItemInsert[] data) returns [string, string][]|error {
+        _ = check self.persistClients.get(ORDER_ITEM).runBatchInsertQuery(data);
+        return from OrderItemInsert inserted in data
+               select [inserted.orderId, inserted.itemId];
+    };
+
+    isolated resource function put orderItems/[string orderId]/[string itemId](OrderItemUpdate data) returns OrderItem|error {
+        _ = check self.persistClients.get(ORDER_ITEM).runUpdateQuery({orderId: orderId, itemId: itemId}, data);
+        return self->/orderItems/[orderId]/[itemId].get();
+    };
+
+    isolated resource function delete orderItems/[string orderId]/[string itemId]() returns OrderItem|error {
+        OrderItem 'object = check self->/orderItems/[orderId]/[itemId].get();
```

Comment:
Any reason to use `'object` as variable name? IMO result will suffice.

Reply:
Fixed in [775f43e](https://github.com/ballerina-platform/module-ballerina-persist/pull/105/commits/775f43eb14f46d103a58633e1e781921f7222a05)